### DISCUSSION TITLE
Running the example HDF5 export and import operations — PyDPF-Core (pyansys.com

### DISCUSSION THREAD

**Discussion by ghost** [ansys/pydpf-core#1187] [Q&A] [answered]
Hi,
I am using python 3.8 and Ansys 2022 R2. I tried to run the example HDF5 export and import operations from the documentation https://dpf.docs.pyansys.com/version/stable/examples/05-file-IO/02-hdf5_serialize_and_read.html#sphx-glr-examples-05-file-io-02-hdf5-serialize-and-read-py
I am hitting the error on line 65 
AttributeError: 'Outputs' object has no attribute 'data_sources'

Could anybody explain why this is happening please

**Comment by PProfizi** [comment, marked as answer]
Hi @Caroline-Poyser,

The example uses the operator hdf5dpf_generate_result_file which only appears in the documentation of PyDPF-Core 0.9.0.
This operator is indeed only available starting with DPF 7.0 (Ansys 2024 R1 pre0).

This example is now marked as requiring this minimal version.

If you wish however to use this feature, what you can do is use DPF Server 2024.1.pre0 which is our standalone version of DPF (to install and use as a Python library), available following the directions [here](https://dpf.docs.pyansys.com/version/stable/user_guide/getting_started_with_dpf_server.html).
